llama_index/evaluation/answer_consistency.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Optional, Sequence
+
+from tonic_validate.classes.benchmark import BenchmarkItem
```

Comment:
Any top-level imports from 3rd party packages (I.e. tonic_validate), should be moved to the functions where they are used. This will help avoid import errors and help CI pass

In the top-level init, you can do a try/except on the import and raise an error if the package is not installed and give a message with the pip command to install.

This is a common pattern for us 👍

Reply:
That makes sense, thanks! I just pushed this change.